Projects/Bounties/README.md

```diff
@@ -52,7 +59,9 @@ The players involved SHOULD marshal the bounty through the following standard st
 ### Complete
 
 1. The Keymaster creates a payment request in the pay server and sends a link directly to the Treasurer.
-1. The Treasurer verifies the request and sends the payment to the Keymaster.
+1. The Treasurer verifies the request then marshals a [1S/1V](../../Process/Voting/Double-Democracy) vote for bounty share-holders to release the funds.
```

Comment:
This is the point where Double Democracy is employed. Each bounty has one or more "shareholders" based on who funded it and how much relatively. The shareholders ultimately decide when the bounty funds are released. The Treasurer is their representative.